### PR TITLE
Fix typos in the rest of the codebase  

### PR DESCRIPTION
**Pull request by luzpaz**
### Description

Fix typos in the rest of the codebase  
Follow up to: 
21cfcec5705ea1c857e7cbdc7363345b7d5cd343  
5d359a857da11ba7404fa674584b600bf84b9e68

Found via `codespell -q 3 -S *.po,*.pot,*.utf8,ChangeLog,./src/external,./plugins/RemoteControl/webroot/js -L anonymus,bu,ded,dout,eyt,fiter,inout,lod,maka,mane,nd,ned,previousy,regio,sav,ser,serie,som,statics,te,tht,ue,unter`

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] This change requires a documentation update
- [X] Housecleaning

### How Has This Been Tested?

n/a

## Checklist:

- [X] My code follows the [code style](http://stellarium.org/doc/head/codingStyle.html) of this project.
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (header file)
- [ ] I have updated the respective chapter in the Stellarium User Guide
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: CREDITS.md [hunk 1]
```diff
@@ -190,7 +190,7 @@
 	       nr_of_measurements, combination of flags, tycho2 number)
 	    -) add all stars from Hipparcos (incl. component solutions), and
 	       tycho2+1st supplement
-	    -) reorganize the stars in several brigthness levels and
+	    -) reorganize the stars in several brigthtness levels and
```
Comment (gzotti) [inline CREDITS.md:193]: brightness

Comment (luzpaz) [reply CREDITS.md:193]: Thanks. Sorry about that. I think added some superfluous whitespace as well. need to go back and fix

File: plugins/Observability/Doxyfile [hunk 2]
```diff
@@ -1241,7 +1241,7 @@ FORMULA_TRANSPARENT    = YES
 USE_MATHJAX            = NO
 
 # When MathJax is enabled you can set the default output format to be used for 
-# thA MathJax output. Supported types are HTML-CSS, NativeMML (i.e. MathML) and 
+# the MathJax output. Supported types are HTML-CSS, NativeMML (i.e. MathML) and
```
Comment (luzpaz) [inline plugins/Observability/Doxyfile:1244]: Please confirm fix

File: scripts/solar_system_screensaver.ssc [hunk 3]
```diff
@@ -1515,7 +1515,8 @@ core.selectObjectByName("Moon", pointer = true);
 StelMovementMgr.setFlagTracking(true);
 
 }
-var pick = pick *( Math.floor(Math.random()*9)+1);
+
+var pick = pick *( Math.floor(Math.random()*9)+1);
```
Comment (luzpaz) [inline scripts/solar_system_screensaver.ssc:1519]: Not sure where these whitespace fixes came from, weird?

Comment (gzotti) [reply scripts/solar_system_screensaver.ssc:1519]: I also wondered. Some syntax checker? I'd prefer to not add empty lines here.

File: plugins/Observability/Doxyfile (hunk 2 above)
Comment (gzotti) [reply plugins/Observability/Doxyfile:1244]: We can safely assume this fix is OK.

**Review by gzotti — CHANGES_REQUESTED** [review]
OK, almost done. I had a few of these on my radar, thanks for collecting them!

**Review by alex-w — CHANGES_REQUESTED** [review]
@luzpaz please resolve finded issues

**Comment by luzpaz** [comment]
Will do so this week

File: scripts/solar_system_screensaver.ssc [hunk 4]
```diff
@@ -1547,7 +1547,8 @@ if (pick == 0)
 {
 core.selectObjectByName("Earth", pointer = true);
 }
-StelMovementMgr.setFlagTracking(true);
+
```
Comment (gzotti) [inline scripts/solar_system_screensaver.ssc:1550]: questionable empty line?

File: scripts/solar_system_screensaver.ssc [hunk 5]
```diff
@@ -2743,7 +2744,8 @@ core.selectObjectByName("Moon", pointer = true);
 StelMovementMgr.setFlagTracking(true);
 
 }
-var pick = pick *( Math.floor(Math.random()*7)+1);
+
```
Comment (gzotti) [inline scripts/solar_system_screensaver.ssc:2747]: questionable empty line?

File: scripts/solar_system_screensaver.ssc [hunk 6]
```diff
@@ -3241,7 +3243,8 @@ core.setDate("2033-07-22T15:36:00", "utc");
 
 core.selectObjectByName("HIP 86742", pointer = true);
 StelMovementMgr.setFlagTracking(true);
-core.selectObjectByName("Mars", pointer = true);
+
```
Comment (gzotti) [inline scripts/solar_system_screensaver.ssc:3246]: questionable empty line?

File: scripts/solar_system_screensaver.ssc [hunk 7]
```diff
@@ -3996,7 +3999,8 @@ core.setDate("2040-01-22T07:00:00", "utc");
 
 core.selectObjectByName("HIP 25606", pointer = true);
 StelMovementMgr.setFlagTracking(true);
-core.selectObjectByName("Mars", pointer = true);
+
```
Comment (gzotti) [inline scripts/solar_system_screensaver.ssc:4002]: questionable empty line?

**Comment by luzpaz** [comment]
Apologies, should be fixed in [bb8bd66](https://github.com/Stellarium/stellarium/pull/2862/commits/bb8bd662cea8cf21a40a039eecb3dd133947b198)

**Review by alex-w — APPROVED** [review]

**Pull request merged**